tests/test_lang_imp/test_request_builder.py

```diff
@@ -98,9 +120,12 @@ class ContextDouble(object):
 
 class TableDouble(object):
 
-    def __init__(self, resolve_values=False, resolve_names=False):
+    def __init__(self, resolve_values=False, resolve_names=False, resolve_headers=False, resolve_header_values=False, resolve_header_names=False):
```

Comment:
I don't think you need to extend this double in this way. You whould be able to use it as it is and eliminate the logic in it.